src/common/settings.js

```diff
@@ -26,6 +26,8 @@ var loadDefault = function(version) {
         showTrayIcon: false,
         trayIconTheme: '',
         disablewebsecurity: true,
+        minimizeToTray: false,
```

Comment:
As discussed above, the default behaviour should be "non" toggling on windows and therefore should be explicitely enabled.